src/SamplesApp/SamplesApp.UITests/SampleControlUITestBase.cs

```diff
@@ -68,7 +68,8 @@ public void BeforeEachTest()
 				}
 			}
 
-			_app = AppInitializer.AttachToApp();
+			var app = AppInitializer.AttachToApp();
+			_app = app ?? _app;
```

Comment:
Not sure to understand this one ?

Reply:
`AppInitializer.AttachToApp()` can return null if the app is already initialized...